lease/lessor.go

```diff
@@ -173,30 +179,40 @@ type lessor struct {
 
 	// Wait duration between lease checkpoints.
 	checkpointInterval time.Duration
+	// the interval to check if the expired lease is revoked
+	expiredLeaseRetryInterval time.Duration
 }
 
 type LessorConfig struct {
-	MinLeaseTTL        int64
-	CheckpointInterval time.Duration
+	MinLeaseTTL                int64
+	CheckpointInterval         time.Duration
+	ExpiredLeasesRetryInterval time.Duration
+}
+
+func (c *LessorConfig) adjust() {
+	if c.CheckpointInterval == 0 {
+		c.CheckpointInterval = defaultLeaseCheckpointInterval
+	}
+	if c.ExpiredLeasesRetryInterval == 0 {
+		c.ExpiredLeasesRetryInterval = defaultExpiredleaseRetryInterval
+	}
 }
 
 func NewLessor(lg *zap.Logger, b backend.Backend, cfg LessorConfig) Lessor {
 	return newLessor(lg, b, cfg)
 }
 
 func newLessor(lg *zap.Logger, b backend.Backend, cfg LessorConfig) *lessor {
-	checkpointInterval := cfg.CheckpointInterval
-	if checkpointInterval == 0 {
-		checkpointInterval = 5 * time.Minute
-	}
+	cfg.adjust()
```

Comment:
Nits:
I think it is a bit confusing for the lessor to change values in its input config. Lessor can choose whether or not to use the values in input config, depending on whether the values are reasonable, but it may not change the config directly. (I know this input config is a just a copy, the code is not doing anything wrong)